plugins/woocommerce/includes/admin/views/html-admin-page-addons.php

```diff
@@ -39,31 +41,35 @@
 	</div>
 
 	<div class="top-bar">
-		<div id="marketplace-current-section-dropdown" class="current-section-dropdown">
-			<ul>
-				<?php foreach ( $sections as $section ) : ?>
+		<ul class="marketplace-header__tabs">
+			<li class="marketplace-header__tab">
+				<a
+					class="marketplace-header__tab-link is-current"
+					href="<?php echo esc_url( admin_url( 'admin.php?page=wc-addons' ) ); ?>"
+				>
+					<?php esc_html_e( 'Browse Extensions', 'woocommerce' ); ?>
+				</a>
+			</li>
+			<li class="marketplace-header__tab">
+				<a
+					class="marketplace-header__tab-link"
+					href="<?php echo esc_url( admin_url( 'admin.php?page=wc-addons&section=helper' ) ); ?>"
+				>
 					<?php
-					if ( $current_section === $section->slug && '_featured' !== $section->slug ) {
-						$current_section_name = $section->label;
-					}
+					$count_html = WC_Helper_Updater::get_updates_count_html();
+					/* translators: %s: WooCommerce.com Subscriptions tab count HTML. */
+					echo ( sprintf( __( 'My Subscriptions %s', 'woocommerce' ), $count_html ) );
 					?>
-					<li>
-						<a
-							class="<?php echo $current_section === $section->slug ? 'current' : ''; ?>"
-							href="<?php echo esc_url( admin_url( 'admin.php?page=wc-addons&section=' . esc_attr( $section->slug ) ) ); ?>">
-							<?php echo esc_html( $section->label ); ?>
-						</a>
-					</li>
-				<?php endforeach; ?>
-			</ul>
-			<div id="marketplace-current-section-name" class="current-section-name"><?php echo esc_html( $current_section_name ); ?></div>
-		</div>
-		</div>
+				</a>
+			</li>
+		</ul>
+	</div>
 
 	<div class="wp-header-end"></div>
 
 	<div class="wrap">
 		<div class="marketplace-content-wrapper">
+			<?php require 'html-admin-page-addons-category-nav.php'; ?>
```

Comment:
Just noting here that other template files prepend `require` paths with the `__DIR__` magic constant.

Reply:
Thanks, added in 54a542bc575c829134da95e745e2a49be2f4f07d.